transport/http/handler_test.go

```diff
@@ -42,6 +42,10 @@ import (
 	"go.uber.org/yarpc/yarpcerrors"
 )
 
+func init() {
```

Comment:
wut.  We can do this at the beginning of tests if we want right?  Why are we depending on init?